src/addTest.test.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @jest-environment jsdom
+ */
+const addTask = require('./addTask.js');
+
+describe('Add new task', () => {
+  test('First test', () => {
+    document.body.innerHTML = '';
+    const task = 'Go to gym';
+    addTask(task);
+    const list = document.querySelectorAll('.item li');
+    expect(list).toHaveLength(1);
+  });
+});
```

Comment:
- [ ] Please use the import statement to test the function.

Reply:
Thanks for the review. I think the above issue is not part of the todo list second milestone test. What do you think?